src/app/core.ts

```diff
@@ -1,6 +1,6 @@
-import {asperaDesktop} from '../index';
 import {messages} from '../constants/messages';
 import {client} from '../helpers/client/client';
+import {safariClient} from '../helpers/client/safari-client';
```

Comment:
Hm any way we don't need to import safari client here? We just did the work to only import `client` above so seems a little off that we now import safari client specifically. Is there a better abstraction? Would it make sense to move any of this stuff to the aspera desktop model where we have `ActivityTracking`?

Reply:
Sure, I cleaned it up a little bit